MultiSource/Benchmarks/Olden/power/build.c

```diff
@@ -10,8 +10,10 @@
  * University of California at Berkeley
  */
 
-#include "power.h"
 #include <stdlib_checked.h>
+#include "power.h"
```

Comment:
You seem to have missed adding `#pragma BOUNDS_CHECKED ON` to the top of `power.h` and turning the pragma off at the bottom. Is this too hard to be viable?